docker/webui/run_openqa.sh

```diff
@@ -7,9 +7,36 @@ else
   /usr/share/openqa/script/upgradedb --user geekotest --upgrade_database
 fi
 
+function scheduler(){
+  su -u /usr/share/openqa/script/openqa-scheduler-daemon
+}
+
+function websockets(){
+  su geekotest -c /usr/share/openqa/script/openqa-websockets-daemon
+}
+
+function gru() {
+  su geekotest -c /usr/share/openqa/script/openqa-gru -m production run
+}
+
+function webui() {
+  su geekotest -c /usr/share/openqa/script/openqa-webui-daemon &
+  apache2ctl start
+  while :; do sleep 100; done;;
```

Comment:
Why is this loop required?

Reply:
Whiout that the container ends. But if we achieve to remove the apache proxy as mentioned in the previous comment, then openqa-webui-daemon could be the main process. I tried to invert the order of apache and the daemon, but didn't work.